examples/react-ts/src/__snapshots__/storyshots.test.ts.snap

```diff
@@ -1,7 +1,7 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Storyshots Demo/AccountForm Standard 1`] = `
-.emotion-15 {
+.emotion-0 {
```

Comment:
those snapshots just have different generated "indices" for those masked class names